patchnotes/4.2.0.md

```diff
@@ -1 +1,32 @@
+## Patchnotes 4.2.0
 
+New Features:
```

Comment:
This however should be a heading IMO.

Personally I'd prefer h3, i.e. `### New Features`

Reply:
good point -> done.